### PR TITLE
Fixing Alerts Tag Issue

### PR DESCRIPTION
**Pull request by kem247**
Tackles #69 in the AllAlerts file under components.

### CODE REVIEW

**Comment by mewtaylor** [comment]
Hey @kem247 I'm seeing some duplicate options in the dropdown:
Screen Shot 2020-11-23 at 11 21 30 AM

I'd take a look and see what built-in js functions might help with turning a list into a set of unique instances - I bet there are some functions in there that could help!

File: src/components/AllAlerts/index.js [hunk 1]
```diff
@@ -213,8 +213,16 @@ class AllAlerts extends Component {
...
+    let uniqueSlice = []
     let sliced = this.state.tags.slice(0, alert.length - 1)
+    sliced.forEach((tags, i) => {
+      if (!uniqueSlice.includes(tags[i])) {
+        uniqueSlice.push(tags[i])
+      }
+    })
+    let stringify = sliced.map(JSON.stringify)
+    let uniqueStringify = new Set(stringify)
+    let uniqueArray = Array.from(uniqueStringify, JSON.parse)
```
Comment (mewtaylor) [inline src/components/AllAlerts/index.js:225]: This does work! I do think there is a simpler way to do this, however. With ES6, we can use the `Set` function, by first flattening sliced, and then grabbing the set, condensing this to a single line:

```js
let uniqueArray = [...new Set(sliced.flat())]
```

Given that this makes it one line, and uses more built-in ES6 functionality, I think it'll be more efficient and clean to do it this way instead. Could you change it to this single line method?

Comment (mewtaylor) [reply src/components/AllAlerts/index.js:225]: @kem247 check out [this answer](https://stackoverflow.com/a/14438954) on stack overflow to see where I got the idea from. I just realized we're getting an array of arrays first though, so applied the `flat()` first which does what you were doing with the stringify process.

Comment (kem247) [reply src/components/AllAlerts/index.js]: Okay, I changed the code to this: 
`    let flattenedArray = sliced.flat()
    let uniqueArray = [...new Set(flattenedArray.flat())]`

**Pull request merged**